src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/create_index_pattern_wizard.tsx

```diff
@@ -31,41 +31,68 @@ import { EmptyState } from './components/empty_state';
 
 import { MAX_SEARCH_SIZE } from './constants';
 import { ensureMinimumTime, getIndices } from './lib';
-import { i18n } from '@kbn/i18n';
+import {
+  SavedObjectsClient,
+  IUiSettingsClient,
+  OverlayStart,
+} from '../../../../../../../../core/public';
+import { DataPublicPluginStart } from '../../../../../../../../plugins/data/public';
+import { IndexPatternCreationConfig } from '../../../../../../../../plugins/index_pattern_management/public';
+import { MatchedIndex } from './types';
+
+interface CreateIndexPatternWizardProps {
+  initialQuery: string;
+  services: {
+    indexPatternCreationType: IndexPatternCreationConfig;
+    es: DataPublicPluginStart['search']['__LEGACY']['esClient'];
+    indexPatterns: DataPublicPluginStart['indexPatterns'];
+    savedObjectsClient: SavedObjectsClient;
+    uiSettings: IUiSettingsClient;
+    changeUrl: (url: string) => void;
+    openConfirm: OverlayStart['openConfirm'];
+  };
+}
+
+interface CreateIndexPatternWizardState {
+  step: number;
+  indexPattern: string;
+  allIndices: MatchedIndex[];
+  remoteClustersExist: boolean;
+  isInitiallyLoadingIndices: boolean;
+  isIncludingSystemIndices: boolean;
+  toasts: EuiGlobalToastListToast[];
+  indexPatternCreationType: IndexPatternCreationConfig;
```

Comment:
I think this can be left as a prop. I don't see where this changes.

Reply:
done.